docs/source/configure/index.rst

```diff
@@ -146,6 +148,54 @@ Environment-specific parameters
   ``"initial_peers" : [{"address":"127.0.0.1:10001", "public_key":
   "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"}]``
 
+Alternative
+===========
+
+Alternative to using a configuration file is setting up the configuration using environment variables.
```

Comment:
strictly speaking, it is not an alternative, as both can be used at once, and your next sentence is just about that